packages/components/pie-button/index.html

```diff
@@ -0,0 +1,11 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <script type="module" src="/src/index.ts"></script>
+  </head>
+  <body>
+    <pie-button />
```

Comment:
probably worth documenting that this is a temporary way to manually test the component